lang/src/test/kotlin/org/partiql/lang/eval/evaluatortestframework/PartiQLCompilerPipelineFactory.kt

```diff
@@ -73,19 +74,22 @@ internal class PartiQLCompilerPipelineFactory : PipelineFactory {
             typedOpBehavior = evaluatorOptions.typedOpBehavior
         )
 
+        @OptIn(ExperimentalWindowFunctions::class)
+        val compiler = if (experimentalFeatures) PartiQLCompilerBuilder.experimental() else PartiQLCompilerBuilder.standard()
```

Comment:
I expect no annotation will be needed here after `experimental()` method is gone.